config/initializers/posts_controller_decorator.rb

```diff
@@ -1,6 +1,13 @@
 Crowdblog::Admin::PostsController.class_eval do
   before_action :load_categories, only: [:edit]
 
+  def index
+    @state = params[:state]
```

Comment:
I'm not sure if the @state variable is needed, I would move it to a helper method in the controller ... or even better, you could work with the Post model and create a scope where you could just pass the state and if none is passed to return all the published_at ...